### ISSUE TITLE
Not all service ports are exposed via minikube service command

### ISSUE THREAD

**Issue by crimeminister** [kubernetes/minikube#659]
BUG REPORT

**Minikube version**

```
v0.10.0
```

**Environment**:
- **OS** (e.g. from /etc/os-release):

```
NAME="Ubuntu"
VERSION="16.04.1 LTS (Xenial Xerus)"
```
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`):

```
kvm
```
- **Docker version**

```
Docker version 1.12.1, build 23cf638
```

**What happened**:

After creating a service that exposes multiple ports, I am only able to access the service via a single IP:port discovered using `minikube service --url <service name>`. It seems as though only the first port listed by the service is mapped.

**What you expected to happen**:

Ideally I would get back a list IP:port pairs, one per exposed service port.

**How to reproduce it**:
Create a service that exposes multiple ports:

```
apiVersion: v1
kind: Service
metadata:
  name: example-service
spec:
  type: NodePort
  ports:
  - name: foo
    port: 8080
    targetPort: foo-port
    protocol: TCP
  - name: bar
    port: 8090
    targetPort: bar-port
    protocol: TCP
  selector:
    app: example
```

Run `minikube service --url`.

**Anything else do we need to know**:

**Comment by r2d4** [comment]
Are you able to access your service on both ports directly, i.e. without the `minikube service command`?

As far as printing out all service ports for `minikube service --url`, that makes sense and it seems like this piece of code is the culprit:

https://github.com/kubernetes/minikube/blob/master/pkg/minikube/cluster/cluster.go#L705

**Comment by crimeminister** [comment]
Yes, I am able to communicate with my service over both ports after figuring out what the corresponding node ports are.

Thanks for highlighting the relevant code! I will have a go at a PR that addresses this, on the off chance that someone else doesn't do it first.

**Comment by r2d4** [comment]
Cool!  Let me know if you run into any problems

**Comment by aaron-prindle** [comment]
Fixed with #729 and #720.  Closing.

**Comment by KIVagant** [comment]
```
$ minikube service docker-registry-docker-registry --format='{{.IP}}:{{.Port}}' --url

Error opening service: Check that minikube is running and that you have specified the correct namespace: parse 192.168.64.4:32275: first path segment in URL cannot contain colon

$ minikube service docker-registry-docker-registry --format='{{.IP}}_{{.Port}}' --url
192.168.64.4_32275
```
